minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterImpl.java

```diff
@@ -600,6 +602,97 @@ public synchronized void start() throws IOException, InterruptedException {
     if (executor == null) {
       executor = Executors.newSingleThreadExecutor();
     }
+
+    verifyUp();
+
+  }
+
+  private void verifyUp() throws InterruptedException, IOException {
+
+    Objects.requireNonNull(getClusterControl().managerProcess, "Error starting Manager");
```

Comment:
Maybe differentiate between the two error conditions?  Guessing on how the difference between the two messages should read.
```
    Objects.requireNonNull(getClusterControl().managerProcess, "Error starting manager - no process");
'''
and
```
Objects.requireNonNull(getClusterControl().managerProcess.info().startInstant().get(),
        "Error starting Manager - instance not started");
```

Reply:
Addressed in 35efd1c